packages/api/src/user/types.ts

```diff
@@ -75,3 +75,13 @@ export interface GetMeResponseData {
 export interface HasBalanceResponseData {
   hasbalance: boolean
 }
+
+export interface ExternalUserArgs {
+  userType: string
+  externalUserID: string
```

Comment:
```suggestion
  externalUserId: string
```
We are already using `Id` for all other args